scoring/static/scoring/scss/mixins.scss

```diff
@@ -133,3 +133,5 @@ $border-priority-3: 1px solid transparentize(#9CBFB7, 0.8);
 @mixin border-priority-3($direction) {
     border-#{$direction}: $border-priority-3;
 }
+
+$border-priority-2-grey:1px solid transparentize($black, 0.8);
```

Comment:
Missing a space after the `:` colon here, and a newline at the end of the file.

…Actually, why is this variable here (in `mixins.scss`) anyway, rather than in `variables.scss`?